src/components/PathwaysList/PathwaysList.tsx

```diff
@@ -38,28 +59,89 @@ const PathwaysList: FC = () => {
 
   const closeImportPathwayModal = useCallback((): void => _setImportPathwayOpen(false), []);
 
+  const handleDelete = useCallback(() => {
+    selected.forEach(id => {
+      deletePathway(id);
+      setSelected(new Set());
+    });
+  }, [deletePathway, selected, setSelected]);
+
+  const [exporting, setExporting] = useState(false);
+
+  const handleExportAll = useCallback(() => {
+    setExporting(true);
+    const zip = new JSZip();
```

Comment:
If there's only one file selected do we want to download as just a json file? The logic can then be expanded for referenced pathways so pathways with references are exported as zips but standalone pathways are json

Reply:
My vote is yes on this.